<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QDialog" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>10</y>
     <width>171</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>New Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Note</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
